recipe/meta.yaml

```diff
@@ -1,42 +1,40 @@
 {% set name = "colour" %}
 {% set version = "0.1.5" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "af20120fefd2afede8b001fbef2ea9da70ad7d49fafdb6489025dae8745c3aee" %}
-{% set build = 0 %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/colour-{{ version }}.tar.gz
+  sha256: af20120fefd2afede8b001fbef2ea9da70ad7d49fafdb6489025dae8745c3aee
 
 build:
   noarch: python
-  number: {{ build }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
You may have to revert this one. I underestimated this package age!

Reply:
was not necessary, just down pinning `setuptools<=67` due to `d2to1` used for packaging. There is an upstream fix, but no reaction by maintainers.